src/python/turicreate/test/test_sframe.py

```diff
@@ -704,6 +704,13 @@ def test_transform(self):
 
         sa = sf.apply(lambda x: x['int_data'] + x['float_data'], float)
         self.__assert_sarray_equal(sf['int_data'] + sf['float_data'], sa)
+    
+    def test_add(self):
+        sf1 = SFrame({'a': [1,2,3]})
+        sf2 = SFrame({'a': [6,7]})
+        sf1=sf1+sf2
+        self.assertTrue({'a': 7} in sf1)
```

Comment:
You probably want a stronger comparison than this. Can you also make the unit test work with 2 columns just in case.

Reply:
@dhivyaaxim - you should also check that `sf1` is exactly what you expect not just that it contains some elements you expect.